clusterman/batch/node_migration.py

```diff
@@ -30,21 +33,24 @@
 from clusterman.config import load_cluster_pool_config
 from clusterman.config import POOL_NAMESPACE
 from clusterman.config import setup_config
+from clusterman.migration.event import MigrationEvent
+from clusterman.migration.settings import WorkerSetup
 from clusterman.util import get_pool_name_list
 from clusterman.util import setup_logging
 
 
 class NodeMigration(BatchDaemon, BatchLoggingMixin, BatchRunningSentinelMixin):
     notify_emails = ["compute-infra@yelp.com"]
 
-    POOL_SCHEDULER = "kubernetes"
+    POOL_SCHEDULER = "kube"
```

Comment:
it should be kubernetes

Reply:
I thought so, then testing got me confused cause it's just `kube` there, I'll get it fixed.